src/main/java/me/zeroeightsix/kami/gui/hudgui/elements/misc/Time.kt

```diff
@@ -2,20 +2,19 @@ package me.zeroeightsix.kami.gui.hudgui.elements.misc
 
 import me.zeroeightsix.kami.event.SafeClientEvent
 import me.zeroeightsix.kami.gui.hudgui.LabelHud
+import me.zeroeightsix.kami.gui.hudgui.elements.combat.CombatItemCount
 import me.zeroeightsix.kami.setting.GuiConfig.setting
 import me.zeroeightsix.kami.util.TimeUtils
 
 object Time : LabelHud(
-    name = "Time",
-    category = Category.MISC,
-    description = "System date and time"
+    category = Category.MISC
 ) {
```

Comment:
I'm not sure why all of these are gone, either

Reply:
Same reason as earlier, they aren't needed and just add extra complication later on , or are just over written.

Reply:
Uh, I don't see how the name and description aren't needed?